stream-product/product-core/src/main/java/com/backbase/stream/product/mapping/ProductMapper.java

```diff
@@ -70,6 +70,7 @@ public interface ProductMapper {
     @Mapping(source = ProductMapperConstants.EXTERNAL_ID, target = ProductMapperConstants.EXTERNAL_ARRANGEMENT_ID)
     @Mapping(source = ProductMapperConstants.PRODUCT_TYPE_EXTERNAL_ID, target = ProductMapperConstants.EXTERNAL_PRODUCT_ID)
     @Mapping(source = ProductMapperConstants.LEGAL_ENTITIES, target = ProductMapperConstants.EXTERNAL_LEGAL_ENTITY_IDS)
+    @Mapping(source = "debitCardsItems", target = "debitCards")
```

Comment:
let's keep the pattern and use constants here

Reply:
@uesleilima Updated with constants, please verify.